src/Symfony/Bundle/WebServerBundle/WebServerConfig.php

```diff
@@ -28,8 +28,8 @@ public function __construct($documentRoot, $env, $address = null, $router = null
             throw new \InvalidArgumentException(sprintf('The document root directory "%s" does not exist.', $documentRoot));
         }
 
-        if (null === $file = $this->guessFrontController($documentRoot, $env)) {
-            throw new \InvalidArgumentException(sprintf('Unable to guess the front controller under "%s".', $documentRoot));
+        if (null === $file = $this->findFrontController($documentRoot, $env)) {
+            throw new \InvalidArgumentException(sprintf('Unable to find the front controller under "%s" (expected to find it as: %s).', $documentRoot, implode(', ', $this->getFrontControllerFileNames($env))));
```

Comment:
Instead of `(expected to find it as: %s)`
why not: `(looked for these files: %s)` or `(none of these files exist: %s)` or something like that.